spec/system/distribution_system_spec.rb

```diff
@@ -9,6 +9,20 @@
     setup_storage_location(@storage_location)
   end
 
+  context "When going to the Pick Ups & Deliveries page" do
+    let(:issued_at) { "2022-08-07 19:00:00".to_datetime }
+    before do
+      item1 = create(:item, value_in_cents: 1050)
+      @distribution = create(:distribution, :with_items, item: item1, agency_rep: "A Person", organization: @user.organization, issued_at: issued_at)
+    end
+
+    it "appears distribution in calendar with correct time & timezone" do
+      visit @url_prefix + "/distributions/schedule"
+      expect(page.find(".fc-event-time")).to have_content "7p"
```

Comment:
To make sure this is a sufficient test, could you confirm that removing the `timeZone: 'UTC'` causes this spec to fail? Thank you!

Reply:
Yes, I tested this scenario before :) 

print spec removing ``` timeZone  'UTC' ```
![image](https://user-images.githubusercontent.com/836472/184497963-69079ec0-49d6-494a-b496-82be76d54e5c.png)